### PR TITLE
Fix enabling experiments

### PR DESCRIPTION
**Pull request by FifiTheBulldog**
Fix the Discord experiments toggle. (Thank you Beef, I used the marked-up snippet here.) The only catch is that disabling experiments only sets the setting; it doesn’t actually disable them until the user reloads.

### CODE REVIEW

File: src/Powercord/plugins/pc-settings/experiments.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  // Unpatch getCurrentUser
+  userMod.getCurrentUser = oldGCUser;
+
+  hasEnabled = true;
+}
+
+async function disableExperiments () {
+  hasEnabled = false;
+}
```
Comment (lexisother) [inline src/Powercord/plugins/pc-settings/experiments.js:47]: Consider showing users a modal with info about a required reload before experiments are fully disabled.

**Review by lexisother — CHANGES_REQUESTED** [review]

**Review by lleyton — APPROVED** [review]
nya

**Comment by MichaelDavidGK** [comment]
image
Discord crashes

**Comment by thennothinghappened** [comment]
what did you do to cause this?

**Comment by MichaelDavidGK** [comment]
> what did you do to cause this?

just updated to latest which required reload

**Comment by thennothinghappened** [comment]
not happening for me. I'll have a look but it's most likely one of your plugins

**Comment by thennothinghappened** [comment]
hmm, not really sure what's going on. that's wrapped in a try catch and is supposed to error silently so it shouldn't be killing it.

**Comment by MichaelDavidGK** [comment]
> not happening for me. I'll have a look but it's most likely one of your plugins

I'm removing all plugins and will be adding one by one to see if that's the issue, will take me a while cause I'm busy with something else rn (15 minutes?)

**Comment by MichaelDavidGK** [comment]
Apparently it's PluginRepo from BD crashing it even tho nothing changed except Replugged update, welp I will live without it tbf

**Comment by thennothinghappened** [comment]
weird, but good that's all cleared up now!

**Comment by asportnoy** [comment]
Reverted this since it caused some issues and didn't work properly for everyone

**Pull request merged**